File: src/api/Controllers/LookupsController.cs
using Microsoft.AspNetCore.Mvc;
using SpaceHabitFrontier.Services;
using SpaceHabitFrontier.Dtos;

namespace SpaceHabitFrontier.Controllers;

[ApiController]
[Route("api/[controller]")]
public class LookupsController : ControllerBase
{

    private readonly ILogger<LookupsController> _logger;
		private readonly LookupsService _lookupsService;

    public LookupsController(
			ILogger<LookupsController> logger,
			LookupsService lookupsService
		)
    {
        _logger = logger;
				_lookupsService = lookupsService;
    }

		[HttpGet(Name = "Lookups")]
    public LookupsDto Get()
    {
			return _lookupsService.GetLookups();
    }

}

File: src/app/src/main/java/space_habit_frontier/app/controllers/LookupsController.java
package space_habit_frontier.app.controllers;

import space_habit_frontier.engine.services.LookupsService;
import space_habit_frontier.engine.dtos.Lookups;

import org.springframework.web.bind.annotation.RestController;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.GetMapping;

@RestController
@RequestMapping("api/lookups")
public class LookupsController {

	private final LookupsService __lookupsService;

	public LookupsController(LookupsService lookupsService) {
		this.__lookupsService = lookupsService;
	}

	@GetMapping
	public Lookups get() {
		return this.__lookupsService.getLookups();
	}

}